CONTRIBUTING.md

```diff
@@ -52,13 +52,13 @@ How to contribute
 
          $ pre-commit run
 
-   To run the `isort` hook alone on changed files:
+   To run the `black` hook alone on changed files:
 
-         $ pre-commit run isort
+         $ pre-commit run black
 
    To run the `isort` hook alone on a given file:
 
-         $ pre-commit run isort <file-path>
+         $ pre-commit run black <file-path>
```

Comment:
do you mean ruff? `--select "D,E,F,I"` will sort imports (if you let ruff `--fix` it)

there is also ruff format now - could remove black:

```
- repo: https://github.com/astral-sh/ruff-pre-commit
  # Ruff version.
  rev: v0.3.1
  hooks:
    # Run the linter.
    - id: ruff
      args: [ --fix ]
    # Run the formatter.
    - id: ruff-format
```

Reply:
Yes, we are slowly moving towards ruff and may be we can use it to get rid of black too but I'd like to test it a bit first to make sure that it has full parity. I'll file an issue.

Reply:
BTW, we already have `ruff` in the pre-commit checks which is why we removed `isort`.

Reply:
Should also be `black` on line 59